.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Argo CD Release
+about: Used by our Release Champion to track progress of a minor release
+title: 'Argo CD Release vX.X'
+labels: 'release'
+assignees: ''
+---
+
+Target RC1 date: ___. __, ____
+Target GA date: ___. __, ____
+
+ - [ ] Create new section in the [Release Planning doc](https://docs.google.com/document/d/1trJIomcgXcfvLw0aYnERrFWfPjQOfYMDJOCh1S8nMBc/edit?usp=sharing)
+ - [ ] Schedule a Release Planning meeting roughly two weeks before the scheduled Release freeze date by adding it to the community calendar (or delegate this task to someone with write access to the community calendar)
```

Comment:
This all makes sense, but is very focused on the end of the cycle. 
Maybe we can add some verbiage about on-going validation as well? 
e.g.
- At the beginning of the release cycle, validate that all features proposed have an assignee and a commitment to complete within the given time window
- At least once during the release (e.g. mid-way through) validate progress and assignees. If assignees have dropped off, or progress is slow, raise concern with leads for possible re-assignments or resource allocation.

Reply:
I like that. @pasha-codefresh since you'll be heading up 2.7, what notes in the issue template would be most helpful for you? Maybe just add the two bullets that Henrik has mentioned?

Reply:
Added notes to the end of the issue template